src/collections/_documentation/error-reporting/quickstart.md

```diff
@@ -41,7 +41,7 @@ After you completed setting up a project in Sentry, you’ll be given a value wh
 
 ## Capturing your first event
 
-Once you have Sentry integrated into your project, you probably want to verify that everything is working as expected before deploying it, and what better way to do that than to break your application!
+Once you have Sentry integrated into your project, you probably want to verify that everything is working as expected before deploying it.
```

Comment:
Not all platforms suggest "breaking your application", therefore move this sentence to where it is appropriate, and leave it out elsewhere.